.github/workflows/single-binary.yml

```diff
@@ -174,7 +174,7 @@ jobs:
       - name: Install Python dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install flytekit flytekitplugins-deck-standard
+          pip install flytekit flytekitplugins-deck-standard 'numpy<2'
           pip freeze
```

Comment:
@eapolinario this change his new. Numpy 2 was released 3 hours ago and flytekit isn't compatible yet which made CI fail:

https://github.com/flyteorg/flyte/actions/runs/9536915121/job/26284633221?pr=4726

In [this](https://github.com/flyteorg/flytekit/pull/2483) PR I fix the error which caused in this CI job and I also add numpy as an explicit flytekit dependency.

Reply:
I'll make a PR to remove numpy here again after the next flytekit release.